lib/_emerge/EbuildPhase.py

```diff
@@ -282,6 +286,9 @@ def _ebuild_exit_unlocked(self, ebuild_process, unlock_task=None):
 			_preinst_bsdflags(settings)
 		elif self.phase == "postinst":
 			_postinst_bsdflags(settings)
+			msg = _postinst_maintainer_nag(settings, self._get_metadata_maintainers())
+			if msg:
+				self._elog('ewarn', [msg], background=True)
```

Comment:
The _elog method is now a coroutine, so the calling function will have to be a coroutine as well. A simple fix would be to move this check to the _PostPhaseCommands _async_start coroutine like this:
```diff
diff --git a/lib/_emerge/EbuildPhase.py b/lib/_emerge/EbuildPhase.py
index fbb040ffd..ff3f93b02 100644
--- a/lib/_emerge/EbuildPhase.py
+++ b/lib/_emerge/EbuildPhase.py
@@ -442,6 +442,10 @@ class _PostPhaseCommands(CompositeTask):
 	__slots__ = ("commands", "elog", "fd_pipes", "logfile", "phase", "settings")
 
 	def _start(self):
+		self.scheduler.run_until_complete(self._async_start())
+
+	@coroutine
+	def _async_start(self):
 		if isinstance(self.commands, list):
 			cmds = [({}, self.commands)]
 		else:
@@ -451,6 +455,11 @@ class _PostPhaseCommands(CompositeTask):
 			cmds = [(kwargs, commands) for kwargs, commands in
 				cmds if not kwargs.get('selinux_only')]
 
+		if self.phase == "postinst":
+			msg = _postinst_maintainer_nag(self.settings, self._get_metadata_maintainers())
+			if msg:
+				yield self.elog('ewarn', [msg], background=True)
+
 		tasks = TaskSequence()
 		for kwargs, commands in cmds:
 			# Select args intended for MiscFunctionsProcess.

```

Reply:
The existing EbuildPhase _async_start method would also work, but anyway maybe it makes more sense to show this around the time of package.mask messages, so it shows for emerge --pretend.